libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Util/SFManagedPreferences.m

```diff
@@ -84,55 +86,56 @@ - (id)init {
         if (self.rawPreferences) {
             [[SalesforceSDKManager sharedManager] registerAppFeature:kSFAppFeatureManagedByMDM];
         }
-        NSMutableDictionary *attributes = [[NSMutableDictionary alloc] init];
-        if (self.rawPreferences) {
-            attributes[@"mdmIsActive"] = [NSNumber numberWithBool:YES];
-            attributes[@"mdmConfigs"] = self.rawPreferences;
-        } else {
-            attributes[@"mdmIsActive"] = [NSNumber numberWithBool:NO];
-        }
-        [SFSDKEventBuilderHelper createAndStoreEvent:@"mdmConfiguration" userAccount:nil className:NSStringFromClass([self class]) attributes:attributes];
```

Comment:
Or technically, making the new `storeAnalyticsEvent` call there (I presume we'd still need the refactor out into that method.